primitives/consensus/babe/src/lib.rs

```diff
@@ -94,9 +96,9 @@ pub type BabeAuthorityWeight = u64;
 /// of 0 (regardless of whether they are plain or vrf secondary blocks).
 pub type BabeBlockWeight = u32;
 
-/// Make a VRF transcript data container
-pub fn make_transcript(randomness: &Randomness, slot: Slot, epoch: u64) -> VrfTranscript {
-	VrfTranscript::new(
+/// Make VRF input suitable for BABE's randomness generation.
+pub fn make_vrf_transcript(randomness: &Randomness, slot: Slot, epoch: u64) -> VrfInput {
```

Comment:
Do we need to keep this function? Can we just left `make_vrf_sign_data`?

The reason for my doubt is following question (maybe my understanding is wrong): isn't term _transcript_ kinda internal implementation detail of sr25519 vrf module?

Reply:
Yes the term transcript IMO is internal and the user (as I explained to jeff) IMO doesn't care about what technically a transcript is and the fiat-shamir transform stuff.

But right not the transcript word is spread around and I don't want to rename too much stuff in this PR.

I still provide the `make_vrf_transcript` alone because in a follow up PR we can let BABE generate the vrf output before sign (as I explained in prev comment). Doesn't make sense to sign if the score is above the threshold and thus you don't have the right to author the block

Reply:
I see. If possible hiding `transcript` would make things more readable in the future.